airbyte-integrations/connectors/source-iterable/source_iterable/api.py

```diff
@@ -142,41 +162,49 @@ def path(self, **kwargs) -> str:
 class EmailBounce(IterableExportStream):
     name = "email_bounce"
     data_field = "emailBounce"
+    cursor_field = "createdAt"
 
 
 class EmailClick(IterableExportStream):
     name = "email_click"
     data_field = "emailClick"
+    cursor_field = "createdAt"
 
 
 class EmailComplaint(IterableExportStream):
     name = "email_complaint"
     data_field = "emailComplaint"
+    cursor_field = "createdAt"
 
 
 class EmailOpen(IterableExportStream):
     name = "email_open"
     data_field = "emailOpen"
+    cursor_field = "createdAt"
 
 
 class EmailSend(IterableExportStream):
     name = "email_send"
     data_field = "emailSend"
+    cursor_field = "createdAt"
 
 
 class EmailSendSkip(IterableExportStream):
     name = "email_send_skip"
     data_field = "emailSendSkip"
+    cursor_field = "createdAt"
```

Comment:
maybe nice default in the base class?

Reply:
I was just choosing between the default value and the explicit definition in each class. It seemed to me that it would be more convenient to specify explicitly for developers. But now I see that the same value is repeated many times, so you may be right.